### ISSUE TITLE
V10 compatibility for module.json

### ISSUE THREAD

**Issue by itamarcu** [ruipin/fvtt-lib-wrapper#67]
Please update the `module.json` file to specify foundry v10 compatibility!

https://foundryvtt.com/article/manifest-migration-guide/

It would help other module developers, who use lib-wrapper as dependency, and are currently seeing warnings about the module.

**Comment by ruipin** [comment]
The JSON file already specifies compatibility with v10 since months ago.

I will not be using the new manifest format as I do not want to drop compatibility with older Foundry, and was promised by the Foundry team this was fine and did not cause any warnings.

**Comment by itamarcu** [comment]
![image](https://user-images.githubusercontent.com/6516621/183246927-7afb3757-2666-4145-bb37-5efbd45e9a68.png)

Correction:  I think it's fine, the warning is only about "compatibility unknown" which I suspect is due to not having a "verified" version in the manifest - as in, your package is compatible with v10, it just doesn't explicitly say it was verified as working in v10.

This is a minor issue, not a big problem.

However, you should know that you can still maintain compatibility with older foundry versions.  As the migration article above mentions, you can just add the new `compatibility` field without removing the existing fields in the JSON, like this:


```
{
...
  "minimumCoreVersion": "0.6.5",
  "compatibleCoreVersion": "10",
  "compatibility": {
    "minimum": "0.6.5",
    "verified": "10"
  },
...
}
```

**Comment by ruipin** [comment]
Resolved in [v1.12.7.1](https://github.com/ruipin/fvtt-lib-wrapper/releases/tag/v1.12.7.1)/
